src/client/styles/components/_bookItem.scss

```diff
@@ -38,6 +38,7 @@ $book-card-largescreen-gutter: 3.5%;
     display: block;
     max-width: 30%;
     float: left;
+    width: 30%;
```

Comment:
Set the fix width as placeholder so when the image is loaded the layout won't shift.